samples/matter/common/src/dfu/smp/dfu_over_smp.cpp

```diff
@@ -106,6 +107,7 @@ void DFUOverSMP::Init()
 
 void DFUOverSMP::ConfirmNewImage()
```

Comment:
Just to my knowledge is any form of confirmation required in the SUIT?

Reply:
No, SUIT can verify new images using its semantics and digests, but we can use it to confirm it additionally in the application if needed, or if some users want to confirm something in their application.